pkgs/tools/misc/vsc-leetcode-cli/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildNpmPackage, fetchFromGitHub, lib }:
+
+buildNpmPackage {
+  pname = "vsc-leetcode-cli";
+  version = "unstable-2021-04-11";
```

Comment:
Why don't we use the most recent commit and call it 2.8.1. Pretty sure that corresponds to what's on NPM.

Reply:
Because it isn't 2.8.1? Seems like 2.8.1 was published to NPM but not pushed to GitHub. The package.json on GitHub still has version 2.8.0 whereas NPM has ... 2.8.1. There may be no other differences and I don't care to diff it but even that one change is enough of a distinguishing characteristic for me.

Reply:
Oh you're right.